web/src/settings_invites.js

```diff
@@ -178,6 +184,48 @@ export function set_up(initialize_event_handlers = true) {
     });
 }
 
+function get_invitation_data() {
+    const invite_as = Number.parseInt($("#invite_as").val(), 10);
+    const stream_ids = [];
+
+    $("#invite-stream-checkboxes input:checked").each(function () {
+        const stream_id = Number.parseInt($(this).val(), 10);
+        stream_ids.push(stream_id);
+    });
+
+    const data = {
+        csrfmiddlewaretoken: $('input[name="csrfmiddlewaretoken"]').attr("value"),
+        invite_as,
+        stream_ids: JSON.stringify(stream_ids),
+    };
+    return data;
+}
+
+function submit_invitation_form(invite_id) {
+    const $invite_status = $("#dialog_error");
+    const data = get_invitation_data(invite_id);
```

Comment:
There is no need to pass `invite_id` parameter here.

Reply:
I used `meta.invite` to store the `multiuse_invite` that is being edited currently.

Is this approach alright?